sanic/request.py

```diff
@@ -53,6 +56,38 @@
     from json import loads as json_loads  # type: ignore
 
 
+@dataclasses.dataclass()
+class Credentials:
+    _auth_type: Optional[str]
```

Comment:
Why is `_auth_type` private? Why `Optional`?

Reply:
Because I think no one is really interested in the type of auth. Everyone just need the result.

Reply:
But I can change it to public if you think it's right

Reply:
I'd make it public. And required.